### ISSUE TITLE
cplaced (avs+)

### ISSUE THREAD

**Issue by TbtBI** [EleonoreMizo/fmtconv#24]
Hi,

It seems cplaced accepts the cplaces value and it cannot be changed.
```
Colorbars(pixel_type="yv12")
a=fmtc_resample(width/2, height/2, cplace="mpeg1", cplaces="mpeg1")
fmtc_resample(width/2, height/2, cplaces="mpeg1", cplaced="mpeg2")
Compare(a)
```

**Comment by EleonoreMizo** [comment]
Indeed. It was a copy/paste error, as often. Fixed in 9681b3e14cf15f41dfac2515d188af2e8587efe5.

**Comment by TbtBI** [comment]
Thanks.
